yt/geometry/coordinates/cylindrical_coordinates.py

```diff
@@ -21,8 +21,9 @@
 
 class CylindricalCoordinateHandler(CoordinateHandler):
     name = "cylindrical"
+    _default_axis_order = ("r", "z", "theta")
 
-    def __init__(self, ds, ordering=("r", "z", "theta")):
+    def __init__(self, ds, ordering=None):
```

Comment:
At a minimum, mypy checks that the default values are compatible with the type annotations in the base class, but repeating the annotations wouldn't hurt, indeed. The reason I avoid adding annotations where I don't strictly need them is that we have mypy setup such that it skips inspecting the body of functions that don't have any hints in their signatures, so that every time we add a first hint, we enable checking the body... and potentially reveal some more mistakes that then need fixing to make mypy happy. Since the PR was already quite sizeable, I wanted to keep it as self-contained as possible.